### DISCUSSION TITLE
MindWipe Capitan-Skipper Abuse

### DISCUSSION THREAD

**Discussion by govnodrist** [FakeFishGames/Barotrauma#15166] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

you can get a lot of talent points for whole crew with enough mindwipe

### Reproduction steps

1.capitan reaches 8 level maxed out skipper tree(final talent give him 9 talent point)
2.mindwipe(first time)
3.you get 9 points back
4.same tree(you now have 10!!! points)
5.mindwipe(second time -1 point)
6.same tree(you now have 9 points + from final talent)
7.loop until mindwipes end
8.???
9.profit

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Single player

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Duplicate of https://github.com/FakeFishGames/Barotrauma/discussions/14988
